packages/browser.smash

```diff
@@ -152,6 +152,10 @@
         await browser.driver.actions({bridge: true}).move({origin: await $(element, true)}).perform();
     }
 
+    * Move over {{element}} to {{x}}, {{y}} {
```

Comment:
So this moves the mouse to the given element's position, then moves it x and y pixels over. Wonder if we can come up with a more descriptive name, like `Move mouse from {{element}} by {{x}}, {{y}}`?

Reply:
Yes that makes sense, done :)